riotfile.py

```diff
@@ -408,14 +408,14 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                         Venv(
                             pys="3.9",
                             pkgs={
-                                "gevent": ["~=20.9.0", "~=20.12.0", "~=21.1.0"],
+                                "gevent": ["~=21.1.0", latest],
                                 "greenlet": "~=1.0",
                             },
                         ),
                         Venv(
                             pys="3.10",
                             pkgs={
-                                "gevent": ["~=21.8.0"],
+                                "gevent": ["~=21.8.0", latest],
```

Comment:
we cannot combine this venv with the previous because 21.8.0 is when 3.10 support was added?